API-1230: Update documentation to get updated attributes since a datetime (#398)
akeneo/pim-api-docs: content/rest-api/filter.md

  ]
}
```

## Filter attributes

> When requesting attributes, you can apply filters to get only the ones you want.

Reviewer: ```suggestion
When you request specific attributes, you can use filters to select only the ones you want. 
```

### By update date
::: availability versions=Serenity editions=CE,EE

You can filter the attributes by their update date.

> Below is the operator allowed to filter on this property, as well as the corresponding type of value expected in the `search` query parameter.

Reviewer: ```suggestion
Below you will find the operator allowed to filter on this property, as well as the corresponding type of value required in the `search` query parameter.
```

:::info

> Note that dates should follow the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601).

Reviewer: ```suggestion
Please note that dates should follow the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601).
```

> Please note that dates should follow the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601).

Reviewer 2: ```suggestion
Please note that dates must follow the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601).
```

:::

| Operator | Allowed value type | Filter description |
| ----------------- | -------------- | ------------------ |
| `>` | datetime <br> _Format: ISO 8601_ | Only returns attributes that were <br> updated after the given day and hour |

#### Example

> To get the attributes that were updated since the 4th of July 2020 at 10am (UTC), you can use the following URL.

Reviewer: ```suggestion
To get the attributes that have been updated since July 4th, 2020 at 10 am (UTC), you can use the following URL.
```

When you request specific attributes, you can use filters to select only the ones you want. 

### By attribute codes
::: availability versions=Serenity editions=CE,EE

You can filter the attributes by their code.

Below you will find the operator allowed to filter on this property, as well as the corresponding type of value required in the `search` query parameter.

| Operator | Allowed value type | Filter description |
| ----------------- | -------------- | ------------------ |
| `IN` | an array of existing attribute codes | Only returns attributes that are in the list |

#### Example
```
/api/rest/v1/attributes?search={"code":[{"operator":"IN","value":["code1","code2"]}]}
```

### By update date
::: availability versions=Serenity editions=CE,EE

You can filter the attributes by their update date.

Below you will find the operator allowed to filter on this property, as well as the corresponding type of value required in the `search` query parameter.

:::info
Please note that dates must follow the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601).
:::

| Operator | Allowed value type | Filter description |
| ----------------- | -------------- | ------------------ |

> | `>` | datetime <br> _Format: ISO 8601_ | Only returns attributes that were <br> updated after the given day and hour |

Reviewer 3: only the ISO 8601 format with “:”, "-”  and the Z works. 2020-07-23T15:19:32+00:00 and 20200723T151932Z don’t work.
Maybe it's worth adding it to the documentation.

> | `>` | datetime <br> _Format: ISO 8601_ | Only returns attributes that were <br> updated after the given day and hour |

Reviewer 3: if I update brand attribute option => the request doesn’t return the brand attribute as updated. But if I update an attribute property => the request returns the brand attribute as updated. Maybe it's worth adding it to the documentation.
